### PR TITLE
flow: use metadata capture length as outerLength

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

File: flow/flow.go [hunk 1]
```diff
@@ -755,7 +761,7 @@ func PacketSeqFromGoPacket(packet *gopacket.Packet, outerLength int64, bpf *BPF)
 
 	if len(ps.Packets) > 0 {
 		p := gopacket.NewPacket(packetData[start:], topLayer.LayerType(), gopacket.NoCopy)
-		if metadata := (*packet).Metadata(); metadata != nil {
+		if metadata != nil {
 			p.Metadata().CaptureInfo = metadata.CaptureInfo
 		}
 		ps.Packets = append(ps.Packets, Packet{gopacket: &p, length: 0})
```
Comment (safchain) [inline flow/flow.go]: This has to be fix too, in order to address inner packet metrics

**Comment by vperelmax** [comment]
I checked it in the kubernetes environment where we had problem with uploading big file, it resolves a bug that we saw there

**Comment by safchain** [comment]
rerun tests

**Comment by safchain** [comment]
ok to test

**Comment by lebauce** [comment]
run tests

**Pull request merged**